### DISCUSSION TITLE
Serial to Ethernet Coordinator. Cannot figure out how to find the right port

### DISCUSSION THREAD

**Discussion by w00master** [Koenkk/zigbee2mqtt#8269] [Devices]
Hey Everyone-

I recently obtained a CC2652p2 (Serial to ethernet) coordinator.  I've been stuck in the "device path" area.  I've tried (originally) going the docker route, and then Home Assistant integration route.  In both cases, though I was able to provide the IP path of my coordinator, zigbee2mqtt just cannot find it.  I"m able to get to the web interface of the coordinator, but despite looking for documentation on how to do this, I'm coming up short.  Most the documentation point to USB configurations, but very little on ethernet/network based coordinators. 

The key is that my coordinator is linked via ethernet in a completely different room, home assistant in a different box, docker container in yet another box.  

When trying to utilize the direct HA integration, it still ends up the same error: 

`Zigbee2MQTT:error 2021-08-05 21:40:43: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2021-08-05 21:40:43: Failed to start zigbee
Zigbee2MQTT:error 2021-08-05 21:40:43: Check https://www.zigbee2mqtt.io/information/FAQ.html#help-zigbee2mqtt-fails-to-start for possible solutions
Zigbee2MQTT:error 2021-08-05 21:40:43: Exiting...
Zigbee2MQTT:error 2021-08-05 21:40:43: Error: Failed to connect to the adapter (Error: SRSP - SYS - ping after 6000ms)
    at ZStackAdapter.start (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/zStackAdapter.ts:102:27)
    at Controller.start (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:123:29)
    at Zigbee.start (/app/lib/zigbee.js:67:27)
    at Controller.start (/app/lib/controller.js:106:27)
    at start (/app/index.js:94:5)`

Not sure what I'm doing wrong.  Why despite being able to navigate to the web interface, that all attempts (docker or HA) have failed in trying to find the coordinator.  I'm clearly missing something as I'm sure I'm missing something obvious.  

Edit:  I've found this bit of documentation:
https://www.zigbee2mqtt.io/how_tos/how_to_connect_to_a_remote_adapter.html

Definitely helpful, but unfortunately it puts me in the same position, namely the port/path of the device:

> Add the following entry, replace /dev/ttyACM0 with the correct path to your adapter.
`20108:raw:0:/dev/ttyACM0:115200 8DATABITS NONE 1STOPBIT`

It's this "correct path" that keeps coming up that I sense is the key. How do I find the path of the device that's linked on ethernet and not on the same box like a usb dongle is for most folks?

**Comment by mercenaruss** [comment]
What's the model of your coordinator?
